src/picknik_ur_multi_arm_config/objectives/move_to_pose.xml

```diff
@@ -1,21 +1,21 @@
-<?xml version="1.0"?>
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Don't blame me for the formatting changes here, it was applied by the UI after saving my changes